bin/impl/redeploy.sh

```diff
@@ -22,7 +22,7 @@ if [ -n "$FLUO_TARBALL_PATH" ]; then
 elif [ -n "$FLUO_TARBALL_REPO" ]; then
   echo "Rebuilding Fluo tarball" 
   cd $FLUO_TARBALL_REPO
-  mvn clean package -DskipTests -Daccumulo.version=$ACCUMULO_VERSION -Dhadoop.version=$HADOOP_VERSION
+  mvn clean install -DskipTests -Daccumulo.version=$ACCUMULO_VERSION -Dhadoop.version=$HADOOP_VERSION
```

Comment:
If there are cases where its detrimental, we can hold off on this.  I'll close the PR.